tests/integration/cartography/intel/github/test_repos.py

```diff
@@ -8,423 +10,398 @@
     """
-    repos_data = cartography.intel.github.repos.transform(
-        GET_REPOS,
-        DIRECT_COLLABORATORS,
-        OUTSIDE_COLLABORATORS,
-    )
-    cartography.intel.github.repos.load_github_repos(
-        neo4j_session,
-        TEST_UPDATE_TAG,
-        repos_data["repos"],
-    )
-    nodes = neo4j_session.run(
-        "MATCH(repo:GitHubRepository) RETURN repo.id",
-    )
-    actual_nodes = {n["repo.id"] for n in nodes}
-    expected_nodes = {
-        "https://github.com/simpsoncorp/sample_repo",
-        "https://github.com/simpsoncorp/SampleRepo2",
-        "https://github.com/cartography-cncf/cartography",
+    # Test repository nodes exist
+    expected_repo_nodes = {
+        ("https://github.com/simpsoncorp/sample_repo",),
+        ("https://github.com/simpsoncorp/SampleRepo2",),
+        ("https://github.com/cartography-cncf/cartography",),
+        ("https://github.com/johndoe/personal-repo",),
+        ("https://github.com/simpsoncorp/another-repo",),
     }
-    assert actual_nodes == expected_nodes
-
+    actual_repo_nodes = check_nodes(neo4j_session, "GitHubRepository", ["id"])
+    assert actual_repo_nodes == expected_repo_nodes
 
-def test_transform_and_load_repository_owners(neo4j_session):
-    """
-    Ensure we can transform and load GitHub repository owner nodes.
-    """
-    repos_data = cartography.intel.github.repos.transform(
-        GET_REPOS,
-        DIRECT_COLLABORATORS,
-        OUTSIDE_COLLABORATORS,
-    )
-    cartography.intel.github.repos.load_github_owners(
-        neo4j_session,
-        TEST_UPDATE_TAG,
-        repos_data["repo_owners"],
+    # Test all core repository properties are populated (no None values)
+    core_properties = [
+        "id",
+        "name",
+        "fullname",
+        "description",
+        "primarylanguage",
+        "homepage",
+        "defaultbranch",
+        "defaultbranchid",
+        "private",
+        "disabled",
+        "archived",
+        "locked",
+        "giturl",
+        "url",
+        "sshurl",
+        "createdat",
+        "updatedat",
+    ]
+    repo_properties = check_nodes(neo4j_session, "GitHubRepository", core_properties)
+    assert repo_properties is not None
+    assert len(repo_properties) == 5
+
+    # Verify no essential properties are None
+    for repo_data in repo_properties:
+        essential_props = repo_data[:17]  # All properties except optional ones
+        assert all(
+            prop is not None for prop in essential_props[:1]
+        ), "ID should not be None"
+        assert all(
+            prop is not None for prop in essential_props[1:7]
+        ), "Core text properties should not be None"
+        assert all(
+            isinstance(prop, bool) for prop in essential_props[8:12]
+        ), "Boolean flags should be boolean"
+        assert all(
+            prop is not None for prop in essential_props[12:17]
+        ), "URL and date properties should not be None"
```

Comment:
Not necessary. Integ tests in cartography mostly only need to test for nodes being attached to each other. All of this just makes the test unnecessarily long and hard to read.